adapters/acuityads/acuityads.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 type AcuityAdsAdapter struct {
-	endpoint template.Template
+	endpoint *template.Template
```

Comment:
Why did this become a pointer?

Reply:
The method to render a template is a pointer receiver and much of the internal data structures to a Template are also pointers. We were converting the value to a pointer right before rendering, so this is a minor improvement to avoid the unnecessary copies and simplifies the template rendering code. I don't expect this to be a problem since the Go docs specifically state: 

> a template may be executed safely in parallel